state-chain/pallets/cf-elections/src/electoral_systems/tests/utils.rs

```diff
@@ -0,0 +1,27 @@
+use cf_primitives::AuthorityCount;
+
+use crate::{
+	electoral_system::{ConsensusVote, ConsensusVotes, ElectoralSystem},
+	vote_storage::VoteStorage,
+};
+
+// Used in the unsafe_median and monotonic_median tests.
+pub fn generate_votes<ES>(
```

Comment:
Thought about it, but they do slightly different things, and I think it makes sense for the tests for the electoral systems which depend on the votes to define the votes themselves.

Reply:
Sure but they don't need to define *how* to build the votes struct.

Reply:
We could create an `into_consensus_votes` which would then force people to provide a validator id and properties for example, if we want it to be general for all - which makes it messier to consume within the tests... without providing another wrapper per test to do it... at which point we're not really achieving too much by pulling anything out

Reply:
We can have a single method that allows all options and simplified methods for the narrower use cases. Then at least we know all the tests are being set up consistently rather than every test implementing its own method. 

Or just make sure that the 'everything' method accepts Default::default. (Can also be made nicer to use with a struct, then you can call, say, 

```
generate_votes(TestVotes {
	num_votes: 10,
	..Default::default()
})
```

Anyway, not urgent right now.